### DISCUSSION TITLE
После обновы zapret пропал в GUI openwrt service zapret

### DISCUSSION THREAD

**Discussion by Psalmir** [bol-van/zapret#999] [Q&A] [answered]
Всем привет, стояла версия zapret v68** но на ней стал ютуб работать не стабильно, решил обновить на последнюю версию v69**  , ютуб заработал отлично никаких проблем нет, но вот в GUI openwrt пропал service zapret
![image](https://github.com/user-attachments/assets/973ea4c9-f5d7-425a-9e73-1a3102070583)
а мне иногда нужно заходить в веб морду роутера и стопать сервис zapret.
Кто нибудь знает как можно вернуть эту функцию?

**Comment by numbereleven-a** [comment, marked as answer]
у болвана нет gui, обновляйте его откуда качали
либо заходите в консоль и командой `/etc/init.d/zapret restart`
